src/Api/Auth/Controllers/AccountsController.cs

```diff
@@ -255,6 +255,23 @@ public async Task PostVerifyEmail()
         await _userService.SendEmailVerificationAsync(user);
     }
 
+    [HttpPost("request-sm-access")]
+    public async Task RequestSMAccessFromAdmins([FromBody] RequestSMAccessRequestModel model)
+    {
+        if (model?.EmailContent == null || model?.OrganizationId == null)
+        {
+            throw new BadRequestException();
+        }
+
+        var user = await _userService.GetUserByPrincipalAsync(User);
+        if (user == null)
+        {
+            throw new UnauthorizedAccessException();
+        }
+
+        await _userService.SendRequestAccessToSM(Guid.Parse(model.OrganizationId), user, model.EmailContent);
+    }
```

Comment:
Same here, since this is a Secrets Manager specific feature, we should own it. 

I propose we make a new controller under `src/Api/SecretsManager/Controllers`

Reply:
I've named it RequestSMAccessController.cs however I am open to better names 😆